src/S3/CMakeLists.txt

```diff
@@ -18,6 +18,9 @@ else()
         /wd4996
     )
 endif()
+if(DETHRACE_FIX_BUGS)
```

Comment:
Not in this bug report, but I think we should combine `BRENDER_FIX_BUGS` and `DETHRACE_FIX_BUGS` in one cmake option + compile definition.